src/openms/source/CHEMISTRY/MASSDECOMPOSITION/IMS/IMSIsotopeDistribution.cpp

```diff
@@ -59,220 +55,212 @@ IsotopeDistribution::IsotopeDistribution(mass_type mass): nominalMass(0) {
 }
 */
 
-    IMSIsotopeDistribution & IMSIsotopeDistribution::operator=(const IMSIsotopeDistribution & distribution)
+  IMSIsotopeDistribution & IMSIsotopeDistribution::operator=(const IMSIsotopeDistribution & distribution)
```

Comment:
did you check if it can be defaulted? (= default) I dind't but could be a candidate